gfosd/components/aggregate.py

```diff
@@ -12,11 +12,7 @@ def __init__(self, component_list, *args, **kwargs):
     def prepare_attributes(self, T, p=1):
         helper_removed = False
         for ix, c in enumerate(self._gf_list):
-            if not helper_removed and c._diff == 0:
```

Comment:
Some important (but not documented) logic was broken here, I think. We need to catch the cases where there are more than one helper variable, and we can only reduce the problem size for one of the helper variables. I'll dig into this.